runtime/util/shchelp_j9.c

```diff
@@ -152,15 +160,26 @@ isLambdaClassName(const char *className, UDATA classNameLength)
 
 #if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
 BOOLEAN
-isLambdaFormClassName(const char *className, UDATA classNameLength)
+isLambdaFormClassName(const char *className, UDATA classNameLength, UDATA *prefixLength)
```

Comment:
Given that `MethodHandles`s, and thus `LambdaForm`s, aren't currently supported in AOT, I don't think `isLambdaFormClassName` needs to be changed.

Reply:
I think it's still worth keeping since we want to support lambda forms eventually.